src/components/Navbar/index.tsx

```diff
@@ -32,18 +53,112 @@ export default function Navbar({ Logo }: INavbarProps) {
   };
   const debounceFn = debounce(handleSearch);
 
-  const filter: FilterCharacter = {
-    name: "rick"
-  };
+  const [search, { loading, data, error }] = useLazyQuery(NavbarSearch);
+
+  useEffect(() => {
+    if (loading === false && data) {
+      setCharacters(data.characters.results);
+      setEpisodes(data.episodes.results);
+      if (
+        data.characters.info.count > data.episodes.info.count ||
+        data.episodes.info.count === null
+      ) {
+        setIsCharacter(true);
+      } else {
+        setIsCharacter(false);
+      }
+      setSearchResultsNames(prevState => {
+        prevState = data.characters.results.map(
+          ({ name }: ICharacter, index: number) => {
+            if (index < 10) {
+              setSearchResultsNames(prevState => [
+                ...prevState,
+                <button
+                  className="navbar-dropdown-item"
+                  onClick={() => {
+                    setSearchValue(name);
+                    setSearchResultsNames([]);
+                  }}
+                >
+                  {name}
+                </button>
+              ]);
+            }
+          }
+        );
+        return prevState;
+      });
 
-  const { error, refetch } = useQuery(NavbarSearch, {
-    variables: {
-      filter: filter
+      //     data.episodes.results.map(({ name }: ICharacter) =>
+      //       setSearchResultsNames(prevState => [
+      //         ...prevState,
+      //         <button
+      //           className="navbar-dropdown-item"
+      //           onClick={() => {
+      //             setSearchValue(name);
+      //             setSearchResultsNames([]);
+      //           }}
+      //         >
+      //           {name}
+      //         </button>
+      //       ])
+      //     );
     }
-  });
+  }, [loading, data]);
+
+  useEffect(() => {
+    if (isCharacter === false) {
+      setEpisodeArray(episodeArray => {
+        episodeArray = episodes.map(
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          ({ id, name, air_date, episode }: IEpisode) => {
+            return (
+              <>
+                <div className="navbar-episodes-item">
+                  <EpisodeCard
+                    key={id}
+                    title={name}
+                    date={air_date}
+                    episode={episode}
+                    description="lorem ipsum"
+                    favorited={false}
+                    handleSetFavorited={() => console.log("here")}
```

Comment:
Remove console logs

Reply:
this will be changed when favoriteContext is introduced